OpenLIFUProtocolConfig/Resources/UI/OpenLIFUProtocolConfig.ui

```diff
@@ -212,7 +223,14 @@
         </widget>
       </item>
       <item>
-        <layout class="QHBoxLayout" name="protocolButtonsLayout">
+        <layout class="QHBoxLayout" name="protocolSaveDeleteButtonsLayout">
+          <item>
+            <widget class="QPushButton" name="protocolFileSaveButton">
```

Comment:
This button is lacking a tooltip that explains what it does. The tooltip is a great opportunity to alleviate any confusion that may arise from having two different types of save buttons. And as with all our buttons: when the button is disabled, the tooltip should explain why it's disabled and/or what can be done to enable it.